termwiz/src/render/terminfo.rs

```diff
@@ -52,32 +52,37 @@ impl TerminfoRenderer {
         allow(clippy::cyclomatic_complexity, clippy::cognitive_complexity)
```

Comment:
I'd be curious to see whether `cargo clippy` is significantly happier about this with your simplification.  I doubt it, but am curious nonetheless!

Reply:
`cargo clippy` doesn't complain about this code if I remove this.  I'll remove it.  (You have other clippy warnings and errors, by the way).